frontend/pages/_error.js

```diff
@@ -26,9 +27,11 @@ const Error = ({ statusCode = 500, message = 'Something went wrong' }) => (
       <p className="is-size-5" style={{ marginBottom: '1rem' }}>
         {message}
       </p>
-      <a className="button is-medium is-primary" href={HOME_ROUTE}>
-        Back to home
-      </a>
+      <Link href={HOME_ROUTE}>
```

Comment:
The link above uses `href="/"`, while this Link uses `HOME_ROUTE`, which is equivalent to `/`. We should standardize on either not using the constant or using it.

Reply:
I'm for replacing everything with HOME_ROUTE, since we could decide to change what route is the landing page in the future.